### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by sanchothefat**
Instructions on how to get up and running locally - cheers for putting this together @joehoyle :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+Node Tachyon
+============
+
+A local Tachyon server to help with development of the main plugin and app:
+
+[WordPress plugin](https://github.com/humanmade/tachyon-plugin)
+
+[Tachyon app](https://github.com/humanmade/tachyon)
```
Comment (joehoyle) [inline README.md]: I don't think we need tahcyon app, that's what `node-tachyon` is replacing :)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Installation
+
+Make sure you have libvips installed, the easiest way is to use homebrew:
+
+```
+brew install homebrew/science/vips --with-webp --with-graphicsmagick
```
Comment (joehoyle) [inline README.md]: Well done for getting it running under OSX! It's a bit easier until linux, just `npm install`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+```
+git clone git@github.com:humanmade/node-tachyon.git
+npm install
+```
+
+## Running the server
+
+```
+node server.js
```
Comment (joehoyle) [inline README.md]: you can also secpficy the port `node server.js 80` for example, and also pass `--debug` for logging.

File: README.md (hunk 1 above)
Comment (sanchothefat) [inline README.md]: Ah yeah, was more for reference I guess, will drop it :)

File: README.md (hunk 3 above)
Comment (sanchothefat) [inline README.md]: Neat, will add that info too

**Comment by joehoyle** [comment]
Fantastic, thanks!

**Pull request merged**